tests/kubeaddons-enterprise/addons/zookeeper/0.x/zk-install-0-x/00-assert.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: zookeeper
```

Comment:
not sure this is the assert we want

Reply:
First asserts are just for addon creations

Reply:
To add more to this pattern of just checking addons, this is coming from default requirments like prometheus etc. Initial idea was to merge yaml to create an addon that doesnt have certain fields. But after a sync with you Ken we decided to go this pattern of install/update to do the tests